api/app.js

```diff
@@ -59,6 +60,10 @@ app.use(
 app.use(logger('dev'));
 app.use(express.urlencoded({ extended: false }));
 app.use(cookieParser());
+//using auth0 middleware to verify user is authenticated
+app.use(authRequired);
+//using authProfile middle ware to save current user data
+app.use(authProfile);
```

Comment:
Really smart way to ensure that the FE receives user info on every request - this means we can probably clean up the FE by removing `axios` calls to the BE specifically for user info if we're careful!